examples/Polls Hypermedia API.md

```diff
@@ -0,0 +1,613 @@
+                                }
+                            ],
+                            "rel": [ "choice" ],
+                            "properties": {
+                                "choice": "Ruby",
+                                "votes": 256
+                            },
+                            "links": [
+                                {
+                                    "rel": [ "self" ],
+                                    "href": "/questions/1/choices/4"
+                                }
+                            ]
+                        }
+                    ]
+                }
+            ]
+        }
+
++ Response 200 (application/hal+json)
+
+        {
+            "_links": {
+                "self": { "href": "/questions" },
+                "next": { "href": "/questions?page=2" }
+            },
+            "_embedded": {
+                "question": [
+                    {
+                        "_links": {
+                            "self": { "self": "/questions/1" }
+                        },
+                        "_embedded": {
+                            "choice": [
+                                {
+                                    "_links": {
+                                        "self": { "self": "/questions/1/choices/1" }
+                                    },
+                                    "choice": "Swift",
+                                    "votes": 2048
+                                }, {
+                                    "_links": {
+                                        "self": { "self": "/questions/1/choices/2" }
+                                    },
+                                    "choice": "Python",
+                                    "votes": 1024
+                                }, {
+                                    "_links": {
+                                        "self": { "self": "/questions/1/choices/3" }
+                                    },
+                                    "choice": "Objective-C",
+                                    "votes": 512
+                                }, {
+                                    "_links": {
+                                        "self": { "self": "/questions/1/choices/4" }
+                                    },
+                                    "choice": "Ruby",
+                                    "votes": 256
+                                }
+                            ]
+                        },
+                        "question": "Favourite programming language?",
+                        "published_at": "2014-11-11T08:40:51.620Z"
+                    }
+                ]
+            }
+        }
+
+### Create a New Question [POST]
+
+You may create your own question using this action. It takes a JSON object containing a question and a collection of answers in the form of choices.
+
+- question (string) - The question
```

Comment:
I still have concerns mixing `-`s with `+`s (as to prevent confusion), but I will leave this up to your judgement @kylef